axonserver/src/main/java/io/axoniq/axonserver/component/ComponentItems.java

```diff
@@ -34,7 +36,7 @@ public ComponentItems(String component, String context, Iterable<I> delegate) {
     public Iterator<I> iterator() {
         return stream(delegate.spliterator(), false)
                 .filter(item -> item.belongsToComponent(component))
-                .filter(item -> item.belongsToContext(context))
+                .filter(item -> StringUtils.isEmpty(context) || context.equals("null") || item.belongsToContext(context))
```

Comment:
when would context be "null"?

Reply:
I adapted to existing behavior, that is if context equals "null" that is considered like "*" or  "all". I started with "all" but since other components like queries and commands stats listing work with "null" I followed that path not to change too many components...